### PR TITLE
Rates type selector aligned

### PR DESCRIPTION
**Pull request by Asizon**
Fixes https://github.com/betaflight/betaflight-configurator/issues/2003

### CODE REVIEW

File: src/css/tabs/pid_tuning.css [hunk 1]
```diff
@@ -260,6 +260,10 @@
     border: 1px solid var(--subtleAccent);
 }
 
+.tab-pid_tuning .select_left_aligned table select {
+    text-align-last: left;
+}
```
Comment (McGiverGim) [inline src/css/tabs/pid_tuning.css:265]: Usually is not a good practice name a CSS style with "what it does", usually is better to name it with "to what it applies". If in this way in a future we decide to change to right aligned we don't need to change the name, only the contents.
There are other selects in this tab that need to be right aligned? If not simply removing the select_left_aligned will apply the style to all the tab.

Comment (Asizon) [reply src/css/tabs/pid_tuning.css:265]: Yes we have got other selects in this tab that depends of right aligment. For the name, i asign that thinking in the future, if we need to add this to other selector that required aligment, the it is already unified

Comment (Asizon) [reply src/css/tabs/pid_tuning.css:265]: So @McGiverGim ,I will apply this new css style to the existing rates_type class ok?

Comment (McGiverGim) [reply src/css/tabs/pid_tuning.css:265]: The question is: why is the reason to have some selects with left align and others with right align?
And use this information to assign the name. For example like `principal_feature` and `secondary_feature`, etc.

Comment (McGiverGim) [reply src/css/tabs/pid_tuning.css:265]: Yes, it's not the perfect solution, but better in this case to apply to the element itself if there is no a classification that we can find for this type or elements that differentiate it.
That's my opinion, maybe others think different 😉

**Pull request merged**